src/Core/Variables/StandardVariableProvider.php

```diff
@@ -255,7 +255,7 @@ public function getAccessorsForPath($propertyPath)
     protected function resolveSubVariableReferences($propertyPath)
     {
         if (strpos($propertyPath, '{') !== false) {
-            preg_match_all('/(\{.*\})/', $propertyPath, $matches);
+            preg_match_all('/(\{.*?\})/', $propertyPath, $matches);
```

Comment:
Would this also work for `<f:render partial="DummyPartial" arguments="{text: '{entity.text_{array1.some{key1}}_{key2}}'}" />` 
I think we can have arbitrary nesting here

Reply:
Probably not, but this likely also didn't work beforehand. The way Fluid handles nesting IIRC is by parsing out the inner part of an expression, then matching that again recursively.
So with the given example (simplified):
`... arguments="{text: '{entity.text_{array1.some{key1}}_{key2}}'}"` -argument syntax match-> `text: '{entity.text_{array1.some{key1}}_{key2}}'` -object/array syntax match-> `entity.text_{array1.some{key1}}_{key2}` -> and here it should fail with greedy as with non greedy matching, because the results are:
```
greedy: array(2) {
  [0]=>
  array(2) {
    [0]=>
    string(18) "{array1.some{key1}"
    [1]=>
    string(6) "{key2}"
  }
  [1]=>
  array(2) {
    [0]=>
    string(18) "{array1.some{key1}"
    [1]=>
    string(6) "{key2}"
  }
}
non-greedy: array(2) {
  [0]=>
  array(1) {
    [0]=>
    string(26) "{array1.some{key1}}_{key2}"
  }
  [1]=>
  array(1) {
    [0]=>
    string(26) "{array1.some{key1}}_{key2}"
  }
}
```
Those are both wrong and will not lead to proper evaluation of the expression. Fluid 3 should probably solve this mess once and for all 🤞